### PR TITLE
relativeTo => uriBaseId

### PR DESCRIPTION
**Pull request by lgolding**
Also, some cleanup from previous PR:

* Add omitted file AnnotatedCodeLocationKind.cs
* Change descriptions of `location.decoratedName` and `location.fullyQualifiedLogicalName` per PR feedback.

### CODE REVIEW

File: src/Sarif/Autogenerated/Location.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ public SarifNodeKind SarifNodeKind
         public PhysicalLocation ResultFile { get; set; }
 
         /// <summary>
-        /// The fully qualified name of the logical location where the analysis tool produced the result. If 'logicalLocationKey' is not specified, this member is can used to retrieve the location logicalLocation from the logicalLocations dictionary, if one exists.
+        /// The human-readable fully qualified name of the logical location where the analysis tool produced the result. If 'logicalLocationKey' is not specified, this member is can used to retrieve the location logicalLocation from the logicalLocations dictionary, if one exists.
```
Comment (lgolding) [inline src/Sarif/Autogenerated/Location.cs]: Description change.

Comment (lgolding) [inline src/Sarif/Autogenerated/Location.cs]: Description change.

File: src/Sarif/Autogenerated/Location.cs [hunk 2]
```diff
@@ -57,7 +57,7 @@ public SarifNodeKind SarifNodeKind
         public string LogicalLocationKey { get; set; }
 
         /// <summary>
-        /// The fully qualified name for the logical location, in the compiler's native format.
+        /// The machine-readable fully qualified name for the logical location where the analysis tool produced the result, such as the mangled function name provided by a C++ compiler that encodes calling convention, return type and other details along with the function name.
```
Comment (lgolding) [inline src/Sarif/Autogenerated/Location.cs]: Description change.

**Comment by michaelcfanning** [comment]
+1

**Pull request merged**